homeassistant/components/climate/__init__.py

```diff
@@ -285,6 +285,9 @@ def state_attributes(self):
             is_aux_heat = self.is_aux_heat_on
             data[ATTR_AUX_HEAT] = STATE_ON if is_aux_heat else STATE_OFF
 
+        if self.current_humidity is not None:
+            data[ATTR_CURRENT_HUMIDITY] = self.current_humidity
```

Comment:
Remove this logic from the supported feature check

Reply:
You mean I should put it before supported feature or remove the variable assignment done in supported feature?